### DISCUSSION TITLE
Make content areas adjustable

### DISCUSSION THREAD

**Discussion by hwillson** [apollographql/apollo-client-devtools#412] [Ideas]
Consider using something like [`react-split-pane`](https://www.npmjs.com/package/react-split-pane) to provide customizable nav / content area re-sizing.

@daniman @jglovier - this won't make it into 3.0, but I'm adding it here so we don't lose sight of it. Thanks for the suggestion!

**Comment by alessbell** [comment]
Resolved with https://github.com/apollographql/apollo-client-devtools/pull/947.
